.circleci/modernize/modernize.py

```diff
@@ -430,6 +430,9 @@ def _run_tests():
         os.path.join(
             root, ".circleci/docker_unified_smoke_unit/smoketest/smoketest.py"
```

Comment:
Once we drop support for Python 2.7, we can also remove modernize. I hope this will be soon, since I have tons of issues with modernize hanging stuck locally.